### PR TITLE
Added sorting to the labels & milestones page

### PR DESCRIPTION
**Pull request by Bwko**
Solves  #10

### CODE REVIEW

File: conf/locale/locale_en-US.ini
Comment (andreynering) [inline conf/locale/locale_en-US.ini]: Use underscore for translation keys:

`closestduedate` -> `closest_due_date`

**Review by andreynering — CHANGES_REQUESTED** [review]
Thank you for the PR.

Made just a single small request of change.

**Comment by Bwko** [comment]
@andreynering I've updated my PR

**Comment by andreynering** [comment]
LGTM

**Review by andreynering — APPROVED** [review]

**Comment by thibaultmeyer** [comment]
LGTM

File: templates/repo/issue/labels.tmpl
Comment (tboerger) [inline templates/repo/issue/labels.tmpl]: Please fix the indentation

**Review by tboerger — CHANGES_REQUESTED** [review]

**Comment by Bwko** [comment]
@tboerger Do you mean the` <!-- Sort --> & </div>?`
Is there a tool to automatically format the Golang template files?

**Comment by tboerger** [comment]
If your editor supports editorconfig it should save it properly

**Comment by Bwko** [comment]
My editor didn't support editorconfig files, I had to add a plugin.
Fixed & rebased

File: templates/repo/issue/milestones.tmpl
Comment (tboerger) [inline templates/repo/issue/milestones.tmpl]: Still some wrong indentation :D

**Comment by Bwko** [comment]
Solved

**Comment by tboerger** [comment]
LGTM, will be merged when we branch the 1.0 release

**Review by tboerger — APPROVED** [review]

**Comment by lunny** [comment]
This maybe needs rebase and resolved the conflicts.

**Comment by Bwko** [comment]
@lunny I can't see the conflict. Only the conflicting file but not which line

**Comment by lunny** [comment]
You can handle the conflict before merging. You have to handle it on command line and your local.

**Comment by Bwko** [comment]
Fixed

**Comment by lunny** [comment]
Great, this PR will be delayed until 1.0 released.

**Comment by lunny** [comment]
Please resolve the conflict and we will merge it.

**Comment by lunny** [comment]
LGTM work

**Pull request merged**
